### ISSUE TITLE
Develop branch ?

### ISSUE THREAD

**Issue by LilTiti** [VeryTastyTomato/GaliDAV#37]
Je sais que ça fait un moment qu'on en parle, encore plus depuis que le créateur nous a lui-même conseillé de le faire, mais est-ce que quelqu'un utilise la version de la branche "develop" d'agendav ? Car j'ai fait mes recherches pour le PDF, et il est vrai que je trouve certaines plus lisibles (comme le fichier "fullcalendar.js" qui est visiblement la base de l'affichage des calendriers sur l'appli, et dont j'aurais besoin pour mon PDF), mais par contre il y a beaucoup de différences, comme par exemple des "use" dans les fichiers de "/src", dont certains "use Symfony..." qui m'intriguent. Enfin bref je pense en effet que ça peut nous faire gagner du temps d'utiliser cette branche mais je veux m'assurer qu'on ne passera pas toute une soirée avant de pouvoir l'utiliser... So quelqu'un a-t-il fait le pas ?

**Comment by flowerkinzy** [comment]
Pas moi. Très honnêtement, je ne le ferai pas aujourd'hui car la bdd ça prend du temps. Je suggère à Simon de le faire vu que son agendav a beugué + il est en contact avec adobo

**Comment by Orni2Thorinque** [comment]
@All: Vous voulez que je passe sur la develop branch?

-----Message d'origine-----
De : "flowerkinzy" <notifications@github.com>
Envoyé : ‎09/‎04/‎2015 19:43
À : "VeryTastyTomato/GaliDAV" <GaliDAV@noreply.github.com>
Objet : Re: [GaliDAV] Develop branch ? (#37)

Pas moi. Très honnêtement, je ne le ferai pas aujourd'hui car la bdd ça prend du temps. Je suggère à Simon de le faire vu que son agendav a beugué + il est en contact avec adobo

**Comment by LilTiti** [comment]
Oui ! Tu es notre testeur ! Ton agendav bug toujours ?

**Comment by Orni2Thorinque** [comment]
Ouai, mais je crois que le soucis vient de mon Apache. Je l'ai explosé en faisant l'adminsys.

-----Message d'origine-----
De : "Etienne Leclercq" <notifications@github.com>
Envoyé : ‎09/‎04/‎2015 20:36
À : "VeryTastyTomato/GaliDAV" <GaliDAV@noreply.github.com>
Cc : "Orni2Thorinque" <Orni2Thorinque@gmail.com>
Objet : Re: [GaliDAV] TODO 10: Utiliser la Develop branch d'Agendav (#37)

Oui ! Tu es notre testeur ! Ton agendav bug toujours ?

**Comment by Orni2Thorinque** [comment]
Alors concernant agendacli dbupdate?
Et aussi les répertoires qui sont required dans son code mais qui sont absents dans la develop branch?

**Comment by flowerkinzy** [comment]
J'ai pas compris, c quoi la question ~~
Plus,est - ce que c' exactement la même démarche d'installation pour la develop branch? P-ê pas...

**Comment by LilTiti** [comment]
Moi je n'ai aucun souvenir des erreurs liées à agendacli dbupdate, je sais juste qu'on a tout le temps une erreur (voir images) mais ce n'est pas la même la que toi je crois...
![erroragdvdbupdate](https://cloud.githubusercontent.com/assets/8625398/7101142/98f88dd0-e049-11e4-8b24-a0a69df2a822.png)
![erroragdvdbupdate2](https://cloud.githubusercontent.com/assets/8625398/7101141/98f7a2b2-e049-11e4-8c15-9a79dd86e6e5.png)

**Comment by LilTiti** [comment]
Moi au final je suis d'avis que, si tu n'as toujours pas réussi à utiliser la version develop, repasse à l'originale... Je ne pense pas qu'on est le temps de consacrer autant de temps pour changer de version... @flowerkinzy je sais pas ce que t'en penses

**Comment by flowerkinzy** [comment]
Oui, je suis d'accord sur le principe. Mais dans ce cas, résoudre le TODO 3 #17 devient crucial

**Comment by Orni2Thorinque** [comment]
Bah j'ai suivi la procédure de Fortier. Jai pas trouvé de procédure particulière dans le Git.

-----Message d'origine-----
De : "flowerkinzy" <notifications@github.com>
Envoyé : ‎11/‎04/‎2015 13:00
À : "VeryTastyTomato/GaliDAV" <GaliDAV@noreply.github.com>
Cc : "Orni2Thorinque" <Orni2Thorinque@gmail.com>
Objet : Re: [GaliDAV] TODO 10: Utiliser la Develop branch d'Agendav (#37)

Oui, je suis d'accord sur le principe. Mais dans ce cas, résoudre le TODO 3 #17 devient crucial

**Comment by flowerkinzy** [comment]
ok dac

**Comment by LilTiti** [comment]
@Orni2Thorinque : repasse à notre version du coup et si tu comprends le dust ça m'intéresse (t'as dit que tu faisais une doc dessus). Car une partie de l'affichage des calendriers est géré également en dust il me semble, pour le PDF ça m'intéresse.
